index.html

```diff
@@ -190,9 +190,26 @@ <h2 class="title">
    
         </section>
 
+        <section class="contact-me">
```

Comment:
- [ ] Kindly consider removing this whitespace below form section as it doesn't exist in the figma design.
![q](https://github.com/migcm06/miguel-portfolio/assets/96683843/d3942060-4ec0-4ee1-a50d-fc2df1a78a21)